src/TestGymMemberTracker.java

```diff
@@ -0,0 +1,101 @@
+// --== Test Gym Member Tracker ==--
```

Comment:
- Change the names to valid strings and change the inserted numbers to longs
- You shouldn't have to create any Member objects, just use the state methods
- Change to use the State1 and Member1 classes, rename those classes